src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/cluster.module.ts

```diff
@@ -7,6 +7,7 @@ import { BsDropdownModule } from 'ngx-bootstrap/dropdown';
 import { ModalModule } from 'ngx-bootstrap/modal';
 import { TabsModule } from 'ngx-bootstrap/tabs';
 
+import { AlertModule, TooltipModule } from 'ngx-bootstrap';
```

Comment:
What's the benefit from using the submodule in imports? The drawback using my IDE would be that I'd have to go through all the imports the IDE created automatically for me and adapt them accordingly, as it doesn't seem that it's capable of doing it automatically.

Reply:
During my cleanup/improvement of unit tests I notice that we were importing the same Object from different modules and that can cause failures during unit testing.

So, since it's the recommended way to do it (according to the official [documentation of ngx-bootstrap](https://valor-software.com/ngx-bootstrap/#/accordion)), I changed all imports to the respective submodules.
After doing that I notice that the building and unit testing duration decreased a few seconds, which is a plus.

In regards to the IDE, VS Code was also creating imports from `ngx-bootstrap`, but after I add it to the blacklist it started importing from submodules. Hope yours does the same.